collection_requirements.rst

```diff
@@ -98,24 +98,24 @@ Modules & Plugins
 
 Documentation
 ~~~~~~~~~~~~~~
-All module and plugin ``DOCUMENTATION`` and ``RETURN`` MUST:
 
-* Use the FQCN for ``M(...)`` and ``- module:`` references of ``seealso`` subsections. See `Linking within module documentation <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-within-module-documentation>`_
-* Use the field ``version_added`` to document the version of the collection for which an option, module or plugin was added.
+All modules and plugins MUST:
 
-  * Use collection version numbers for ``version_added``, and not Ansible version numbers or other unrelated version numbers.
-  * If you for some reason really have to specify version numbers of Ansible or of another collection, you have to provide ``version_added_collection``. We strongly recommend to NOT do this.
-  * Not every option, module or plugin must have ``version_added``. You should use it to mark when new content (modules, plugins, options) were added to the collection. The values are shown in the documentation, and this can be very useful for your users.
+* Have a `DOCUMENTATION <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#documentation-block>`_ block
+* Have an `EXAMPLES <https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#examples-block>`_ block (except where not relevant for the plugin type)
+* Use FQCNs (including ansible.builtin) when referring to modules, plugins and documentation fragments inside and outside the collection
```

Comment:
```suggestion
* Use FQCNs when referring to modules, plugins and documentation fragments inside and outside the collection (including ``ansible.builtin.`` for the listed entities from `Ansible-base <https://github.com/ansible/ansible/tree/devel/lib/ansible>`_)
```

Reply:
@Andersson007 is there a better source for this than the git repo ? What do you mean to be linking to ?

Reply:
@dmsimard sorry for late reply, was extremly busy these days. We could also use https://docs.ansible.com/ansible/devel/collections/all_plugins.html (i didn't find another source), I changed the suggestion. Thanks